rest_sensors.cpp

```diff
@@ -2657,6 +2658,7 @@ void DeRestPluginPrivate::handleSensorEvent(const Event &e)
             if (!(item->needPushSet() || item->needPushChange()))
             {
                 DBG_Printf(DBG_INFO_L2, "discard sensor config push for %s (already pushed)\n", e.what());
+                webSocketServer->flush(); // force transmit send buffer
```

Comment:
I get the intention, but imho it isn't the responsibility of this branch to flush the socket, since obviously there isn't anything to do here. I'd propose to leave the flushing to the places where there's actually something to flush.

Reply:
Again, happy to remove it, but this was copied from handling the `state` and `config` attributes.  Same logic also in lights.

Reply:
Ditched 6 calls to `flush()` in three files.